SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialVirtualWorkerTranslationManager.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
```

Comment:
We always want to put engine includes after GDK includes to avoid hiding missing includes in other headers

Reply:
Moved.